students/srepking/Lesson04/mailroom2.py

```diff
@@ -43,10 +45,15 @@ def report():
         if count > 0:
             average = total / count
         donors_f[name] = [total, count, average]
```

Comment:
I see what you're doing now. I overlooked at it first, this is the first version that's doing zero amount validation 🙂.

Since donation of zero doesn't really make sense as a valid amount in this program, it is best to prevent zero from being stored by adding a validation early on.

When user enters amount you can check that the amount is > 0.

Reply:
Fixed!